vic/drivers/cesm/src/cesm_put_data.c

```diff
@@ -258,7 +258,7 @@ vic_cesm_put_data()
 
```

Comment:
They are the same in `~/vic/vic/drivers/shared_all/src/put_data.c`, but these sign conventions agree with those in VIC 4 in RASM (e.g. these make the coupler fields for VIC 5 in RASM agree with VIC 4 in RASM).